scripts/degenApe.js

```diff
@@ -37,72 +37,31 @@ const allReports = [];
 // Addresses & Contracts
 const governance = "0x4204FDD868FFe0e62F57e6A626F8C9530F7d5AD1";
 const strategist = "0x4204FDD868FFe0e62F57e6A626F8C9530F7d5AD1";
-const controller = "0xdc954e7399e9ADA2661cdddb8D4C19c19E070A8E";
+const controller = "0xe5E231De20C68AabB8D669f87971aE57E2AbF680";
 const timelock = "0x4204FDD868FFe0e62F57e6A626F8C9530F7d5AD1";
 const harvester = ["0x0f571D2625b503BB7C1d2b5655b483a2Fa696fEf"];
 
 const contracts = [
-  // "src/strategies/near/trisolaris/strategoy-tri-$aurora-$near-lp.sol:StrategyTriAuroraNearLp",
-  "src/strategies/near/trisolaris/strategy-tri-near-usdc-lp.sol:StrategyTriNearUsdcLp",
-  // "src/strategies/near/trisolaris/strategy-tri-$near-$usdt-lp.sol:StrategyTriNearUsdtLp"
+  "src/strategies/gnosis/curve/strategy-curve-3pool-lp.sol:StrategyXdaiCurve3CRV"
 ];
 
 const testedStrategies = [
-  "0xE197b88C0C94F6396a66f964dbd6F87F11EF95D4"
 ];
 
-// Functions
-const sleep = async (ms, active = true) => {
-  if (active) {
-    console.log("Sleeping...")
-    return new Promise((resolve) => setTimeout(resolve, ms));
-  }
-};
-
-// Use this for verification of 5 contracts or less
-const fastVerifyContracts = async (strategies) => {
-  // await exec("npx hardhat clean");
-  console.log(`Verifying contracts...`);
-  await Promise.all(strategies.map(async (strategy) => {
-    try {
-      await hre.run("verify:verify", {
-        address: strategy,
-        constructorArguments: [governance, strategist, controller, timelock],
-      });
-    } catch (e) {
-      console.error(e);
-    }
-  }));
-}
-
-// Use this for verificationof 5 contracts or more
-const slowVerifyContracts = async (strategies) => {
-  for (strategy of strategies) {
-    try {
-      await hre.run("verify:verify", {
-        address: strategy,
-        constructorArguments: [governance, strategist, controller, timelock],
-      });
-    } catch (e) {
-      console.error(e);
-    }
-  }
-}
-
 const executeTx = async (calls, tx, fn, ...args) => {
-  await sleep(sleepTime, sleepToggle);
-  // if (!txRefs[tx]) { recall(executeTx, calls, tx, fn, ...args) }
+  let transaction;
+  await sleep(sleepConfigs);
```

Comment:
accessing a global variable this way is kinda yucky. When reading the code I assume the variable is local and passed in. Then i spend time looking for it (at least in github. In vscode I would click on it and see where it was declared). It's better to pass sleepconfigs into the function.